server/server.go

```diff
@@ -61,3 +61,34 @@ func (s *server) Read(ctx context.Context, req *pb.ReadRequest) (*pb.ReadReply,
 	reply := &pb.ReadReply{BytesRead: int64(n), Data: data}
 	return reply, nil
 }
+
+func (s *server) Rename(ctx context.Context, req *pb.RenameRequest) (*pb.RenameReply, error) {
+	dn0, ofn, err := splitDiskAndFile(req.Oldname)
+	if err != nil {
+		log.Printf("server: rename error (%v)", err)
+		return &pb.RenameReply{}, nil
```

Comment:
we have not started to set error actually... there is no error handling for now.